yapapi/log.py

```diff
@@ -170,30 +170,22 @@ def _check_event_type_to_string():
 
 
 def log_event(event: events.Event) -> None:
-    """Log an event in human-readable representation."""
+    """Log `event` with a human-readable description."""
 
     loglevel = logging.DEBUG
 
-    def _format(obj: Any, max_len: int = 200) -> str:
-        # This will also escape control characters, in particular,
-        # newline characters in `obj` will be replaced by r"\n".
-        text = repr(obj)
-        if len(text) > max_len:
-            text = text[: max_len - 3] + "..."
-        return text
```

Comment:
This was useful mainly for shortening looong lines, in the times when command output from providers was returned as  a single big chunk. Now each line of the output (more or less) is a separate event so there's really no need to keep this function.